odoo/addons/base/models/ir_asset.py

```diff
@@ -141,9 +141,8 @@ def _fill_asset_paths(self, bundle, addons, installed, css, js, xml, asset_paths
         if bundle in seen:
             raise Exception("Circular assets bundle declaration: %s" % " > ".join(seen + [bundle]))
 
-        if not root._loaded:
-            root.load_addons()
-            root._loaded = True
+        if root.statics is None:
+            root.load_statics()
```

Comment:
This is a call for `lazy_property`!

Reply:
Same as above, problem is `addons_manifest` and is duty for another PR.